### PR TITLE
feat(combo-box): [WC-2049][WC-2050] Add design mode and page explorer text

### PR DESCRIPTION
**Pull request by gjulivan**
### Pull request type
feature (combobox) : add proper text for design mode and page explorer preview

### Description
For Association type: Association caption
For Enumeration type: [enumeration, Attribute]
For Boolean type: [boolean, Attribute]

image

image

### CODE REVIEW

File: packages/pluggableWidgets/combobox-web/src/Combobox.editorPreview.tsx [hunk 1]
```diff
@@ -6,6 +6,32 @@ import { ClearButton, DownArrow } from "./assets/icons";
...
+            returnVal = (optionsSourceAssociationDataSource as { caption?: string })?.caption;
+            break;
+        case "enumeration":
+            returnVal = `[${optionsSourceType}, ${attributeEnumeration}]`;
+            break;
+        case "boolean":
+            returnVal = `[${optionsSourceType}, ${attributeBoolean}]`;
+            break;
+        default:
+            returnVal = `[${emptyOptionText}]`;
```
Comment (iobuhov) [inline packages/pluggableWidgets/combobox-web/src/Combobox.editorPreview.tsx:29]: Why we have `emptyOptionText` twice? On line 29 and on 32

Comment (gjulivan) [reply packages/pluggableWidgets/combobox-web/src/Combobox.editorPreview.tsx:29]: previously, i thought association caption can be null,
but seems like i can't find the real case.

fixed, removed duplicate code,.

**Review by iobuhov — APPROVED** [review]

**Review by iobuhov — DISMISSED** [review]

**Review by rahmanunver — APPROVED** [review]

File: packages/pluggableWidgets/combobox-web/src/helpers/utils.ts [hunk 2]
```diff
@@ -10,3 +12,26 @@ export function getSelectedCaptionsPlaceholder(selector: MultiSelector, selected
...
+        attributeEnumeration,
+        attributeBoolean,
+        emptyOptionText
+    } = args;
+    const emptyStringFormat = emptyOptionText ? `[${emptyOptionText}]` : "Combo box";
+    switch (optionsSourceType) {
+        case "association":
+            type DsProperty = { caption?: string };
+            const dsProperty: DsProperty = datasource(optionsSourceAssociationDataSource)().property ?? {};
+            return dsProperty.caption || emptyStringFormat;
```
Comment (r0b1n) [inline packages/pluggableWidgets/combobox-web/src/helpers/utils.ts:29]: We can read caption directly by doing `optionsSourceAssociationDataSource.caption`

Comment (gjulivan) [reply packages/pluggableWidgets/combobox-web/src/helpers/utils.ts:29]: fixed

**Review by r0b1n — DISMISSED** [review]

**Review by rahmanunver — DISMISSED** [review]

**Review by iobuhov — APPROVED** [review]

**Pull request merged**